client/js/lounge.js

```diff
@@ -37,6 +37,22 @@ $(function() {
 		"/voice",
 		"/whois"
 	];
+	var handledTypes = [
```

Comment:
I'm not a fan of doing that, tbh. It sounds a bit messy, adds a whole bunch of extra code just for removing indexOf? Is it because of the name? When we get babel in, we can use includes, which is better...